examples/js/exchange-capabilities.js

```diff
@@ -33,24 +33,25 @@ async function main () {
     let emulated = 0
 
     const certified = [
```

Comment:
@pcriadoperez  I don't have all the context here, but do we need this list at all? We have a property `certified` in each exchange.